microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -50,6 +50,8 @@ The Evergreen Runtime is preinstalled onto all Windows 11 devices as a part of t
 <!-- ====================================================================== -->
 ## The Fixed Version runtime distribution mode
```

Comment:
```suggestion
## The Fixed Version (Self-Contained) runtime distribution mode
```
@liminzhu @david-risney @aluhrs13 - change heading, or keep as-is?  
Keep "mode" in heading, or write "model" instead?

Reply:
POR: don't put in heading, only in body